### ISSUE TITLE
Define version at a single location

### ISSUE THREAD

**Issue by p-snft** [oemof/oemof-solph#712]
Currently, the version is defined in the following locations:
* VERSION
* docs/conf.py
* setup.py
* `src/oemof/solph/__init__.py`

It would simplify the release process, if all but one would import from the other one.

**Comment by gnn** [comment]
While it's possible to solve this, it's unnecessarily hard. The main problem is that `VERSION` and `setup.py` live outside of the `src` directory so they shouldn't be accessed by code inside `src` and `setup.py` shouldn't access code inside `src` because that code and it's dependencies are not available before the installation is done, i.e. when `setup.py` starts. You can always manually read in files at specific locations, but I don't think that's a good idea.
Most importantly: there are tools adressing this exact problem. We already have a `.bumpversion.cfg` so we could use [`bumpversion`][0]. As I had problems using `bumpversion`, I opted for [`punch.py`][1] in `oemof.tabular`. At least I think so. Last but not least there's also [`setuptools_scm`][2] which I normally don't use because using it creates a runtime dependency on `setuptool` really fast.
In short: I propose getting rid of `VERSION`, using a tool to manage version numbers during the release and documenting it's usage to simplify the release process.

[0]: https://pypi.org/project/bumpversion/
[1]: https://pypi.org/project/punch.py/
[2]: https://pypi.org/project/setuptools-scm/

**Comment by p-snft** [comment]
Can this bee improved at the dev meeting (https://github.com/oemof/oemof/issues/100)?
